docs/csharp/language-reference/keywords/in-parameter-modifier.md

```diff
@@ -56,7 +56,10 @@ You can't use the `in`, `ref`, and `out` keywords for the following kinds of met
   
 - Iterator methods, which include a [yield return](../../../csharp/language-reference/keywords/yield.md) or `yield break` statement.  
 
-You typically declare `in` arguments to avoid the copy operations necessary for passing arguments by value. This is most useful when arguments are structures or arrays of structures.
+You typically declare `in` arguments to avoid the copy operations necessary for passing arguments by value. This is most useful when arguments are value types such as structures where copy operations are more expensive than passing by reference.
```

Comment:
The more I thought about this, the more I liked your proposed wording. Let's :shipit:

Reply:
Great catch on the example.

I have committed the change.

Thanks for your thorough review and help with this @BillWagner!